inference-engine/src/gna_plugin/backend/am_intel_dnn.hpp

```diff
@@ -389,20 +381,16 @@ class AMIntelDNN {
                                          bool postInitMem);
 
     static void InitMaxpoolComponentPrivate(intel_dnn_component_t &cmp,
-                                            uint32_t num_rows_in,
-                                            uint32_t num_columns_in,
-                                            uint32_t num_rows_out,
-                                            uint32_t num_columns_out,
-                                            uint32_t num_bytes_per_input,
-                                            uint32_t num_bytes_per_output,
-                                            uint32_t num_pool_size,
-                                            uint32_t num_pool_step,
-                                            uint32_t num_pool_stride,
-                                            bool do_sum_not_max,
-                                            float output_scale_factor,
-                                            void *&ptr_inputs,
-                                            void *&ptr_outputs,
-                                            bool postInitMem);
+        std::array<uint32_t, 3> inCHW,
```

Comment:
Why not keep the indentation aligned with other code?

Reply:
restored